includes/class-cocart-session.php

```diff
@@ -113,30 +113,28 @@ public static function load_cart_action() {
 
 			wc_nocache_headers();
 
-			// Check the user is logged in. If true a different cart cannot be loaded so just return.
-			if ( is_user_logged_in() ) {
-				$current_user = wp_get_current_user();
-				$user_id      = $current_user->ID;
-
-				// Compare the user ID with the cart key.
-				if ( $user_id === $cart_key ) {
-					/* translators: %s: cart key */
-					CoCart_Logger::log( sprintf( __( 'Cart key "%s" is already loaded as the user is logged in.', 'cart-rest-api-for-woocommerce' ), $cart_key ), 'errro' );
-				} else {
-					/* translators: %s: cart key */
-					CoCart_Logger::log( sprintf( __( 'Customer is already logged in. Cart key "%s" cannot be loaded into session.', 'cart-rest-api-for-woocommerce' ), $cart_key ), 'error' );
-				}
-				return;
```

Comment:
I think maybe if this else statement was removed and the `return;` was moved up one line we might get a better result.